### PR TITLE
Upgrade image-builder

### PR DESCRIPTION
**Pull request by vignesh-goutham**
*Description of changes:*
Update kube-sigs/image-builder to latest release. All patches were remade, as old patches didnt anymore apply cleanly on the newer version.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jaxesn — CHANGES_REQUESTED** [review]
This looks pretty good.  When we merge this lets be on the look out for the next e2e run just to make sure nothing new pops up from this chage.

Can you add an updating section to the readme for this project?  check out the flux or capi projects for an example

**Review by jaxesn — APPROVED** [review]
/lgtm

File: projects/kubernetes-sigs/image-builder/patches/0004-Create-etc-pki-tls-certs-dir-as-part-of-image-builds.patch [hunk 1]
```diff
@@ -23,11 +23,11 @@ index c3431fbf..83552b2f 100644
 +
 +- name: Ensures /etc/pki/tls/certs dir exists
 +  file: path=/etc/pki/tls/certs state=directory
-+  when: pki.stat.isdir is defined and pki.stat.isdir
-+ 
++  when: pki.stat.isdir is defined and pki.stat.isdi
```
Comment (abhay-krishna) [inline projects/kubernetes-sigs/image-builder/patches/0004-Create-etc-pki-tls-certs-dir-as-part-of-image-builds.patch:26]: Should this have been `isdir`?

Comment (vignesh-goutham) [reply projects/kubernetes-sigs/image-builder/patches/0004-Create-etc-pki-tls-certs-dir-as-part-of-image-builds.patch:26]: ah, good catch

**Pull request merged**